src/main/java/com/example/UserService.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
+
+import java.util.List;
+
+public class userService {
+    private List<String> users;
+
+    // Constructor injection
+    public UserService(List<String> users) {
+        this.users = users;
+    }
```

Comment:
This might be unsafe - we should consider cloning this List instead of an assignment, as it has the potential of being changed externally.